<ui version="4.0" >
 <class>dEditAction</class>
 <widget class="Q3MainWindow" name="dEditAction" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>355</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Action</string>
  </property>
  <widget class="QWidget" name="widget" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>355</height>
    </rect>
   </property>
   <layout class="QGridLayout" >
    <item row="0" column="0" >
     <widget class="QTabWidget" name="tabWidget" >
      <property name="enabled" >
       <bool>true</bool>
      </property>
      <property name="font" >
       <font/>
      </property>
      <property name="tabShape" >
       <enum>QTabWidget::Rounded</enum>
      </property>
      <widget class="QWidget" name="tab" >
       <attribute name="title" >
        <string>Common</string>
       </attribute>
       <layout class="QGridLayout" >
        <item row="2" column="0" >
         <widget class="QLabel" name="textLabel5" >
          <property name="text" >
           <string>Description:</string>
          </property>
          <property name="wordWrap" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="0" >
         <layout class="QGridLayout" >
          <item row="0" column="0" >
           <widget class="QLabel" name="textLabel1" >
            <property name="text" >
             <string>Name</string>
            </property>
            <property name="wordWrap" >
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0" >
           <widget class="QLabel" name="textLabel1_2" >
            <property name="text" >
             <string>Type</string>
            </property>
            <property name="wordWrap" >
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="1" >
           <widget class="QComboBox" name="cbType" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Minimum" hsizetype="Expanding" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <item>
             <property name="text" >
              <string>standart command</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>user script</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>external script</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="0" column="1" >
           <widget class="QLineEdit" name="eName" >
            <property name="enabled" >
             <bool>true</bool>
            </property>
            <property name="minimumSize" >
             <size>
              <width>0</width>
              <height>22</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="3" column="0" >
         <widget class="Q3TextEdit" name="eDescription" />
        </item>
        <item row="1" column="0" >
         <layout class="QHBoxLayout" >
          <item>
           <spacer>
            <property name="orientation" >
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType" >
             <enum>QSizePolicy::Expanding</enum>
            </property>
            <property name="sizeHint" >
             <size>
              <width>91</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="Q3GroupBox" name="groupBox1" >
            <property name="title" >
             <string>Enabled icon</string>
            </property>
            <layout class="QGridLayout" >
             <item row="0" column="2" >
              <widget class="QPushButton" name="bSetActive" >
               <property name="maximumSize" >
                <size>
                 <width>20</width>
                 <height>24</height>
                </size>
               </property>
               <property name="text" >
                <string>...</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0" >
              <widget class="QLabel" name="pActive" >
               <property name="scaledContents" >
                <bool>true</bool>
               </property>
               <property name="wordWrap" >
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item row="0" column="3" >
              <widget class="QPushButton" name="bRemoveActive" >
               <property name="enabled" >
                <bool>false</bool>
               </property>
               <property name="maximumSize" >
                <size>
                 <width>20</width>
                 <height>32767</height>
                </size>
               </property>
               <property name="text" >
                <string>X</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="Q3GroupBox" name="groupBox2" >
            <property name="title" >
             <string>Disabled icon</string>
            </property>
            <layout class="QGridLayout" >
             <item row="0" column="0" >
              <widget class="QLabel" name="pPassive" >
               <property name="scaledContents" >
                <bool>true</bool>
               </property>
               <property name="wordWrap" >
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item row="0" column="2" >
              <widget class="QPushButton" name="bSetPassive" >
               <property name="sizePolicy" >
                <sizepolicy vsizetype="Minimum" hsizetype="Minimum" >
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="maximumSize" >
                <size>
                 <width>20</width>
                 <height>32767</height>
                </size>
               </property>
               <property name="text" >
                <string>...</string>
               </property>
              </widget>
             </item>
             <item row="0" column="3" >
              <widget class="QPushButton" name="bRemovePassive" >
               <property name="enabled" >
                <bool>false</bool>
               </property>
               <property name="sizePolicy" >
                <sizepolicy vsizetype="Minimum" hsizetype="Minimum" >
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="maximumSize" >
                <size>
                 <width>20</width>
                 <height>32767</height>
                </size>
               </property>
               <property name="text" >
                <string>X</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="TabPage" >
       <attribute name="title" >
        <string>Execute</string>
       </attribute>
       <layout class="QGridLayout" >
        <item row="0" column="0" >
         <widget class="Q3WidgetStack" name="wStack" >
          <widget class="QWidget" name="WStackPage" >
           <property name="geometry" >
            <rect>
             <x>0</x>
             <y>0</y>
             <width>100</width>
             <height>30</height>
            </rect>
           </property>
           <layout class="QGridLayout" >
            <item row="0" column="0" >
             <layout class="QGridLayout" >
              <item row="1" column="0" >
               <spacer>
                <property name="orientation" >
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeType" >
                 <enum>QSizePolicy::Expanding</enum>
                </property>
                <property name="sizeHint" >
                 <size>
                  <width>21</width>
                  <height>130</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="0" column="0" >
               <layout class="QGridLayout" >
                <item row="1" column="1" >
                 <widget class="QComboBox" name="cbObject" >
                  <property name="sizePolicy" >
                   <sizepolicy vsizetype="Minimum" hsizetype="Expanding" >
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                 </widget>
                </item>
                <item row="2" column="1" >
                 <widget class="QComboBox" name="cbForm" />
                </item>
                <item row="0" column="0" >
                 <widget class="QLabel" name="lAction" >
                  <property name="text" >
                   <string>Action</string>
                  </property>
                  <property name="wordWrap" >
                   <bool>false</bool>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0" >
                 <widget class="QLabel" name="lObject" >
                  <property name="text" >
                   <string>Object</string>
                  </property>
                  <property name="wordWrap" >
                   <bool>false</bool>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0" >
                 <widget class="QLabel" name="lForm" >
                  <property name="text" >
                   <string>Form</string>
                  </property>
                  <property name="wordWrap" >
                   <bool>false</bool>
                  </property>
                 </widget>
                </item>
                <item row="3" column="1" >
                 <widget class="QLineEdit" name="eParams" >
                  <property name="enabled" >
                   <bool>true</bool>
                  </property>
                  <property name="minimumSize" >
                   <size>
                    <width>0</width>
                    <height>22</height>
                   </size>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1" >
                 <widget class="QComboBox" name="cbAction" >
                  <property name="sizePolicy" >
                   <sizepolicy vsizetype="Minimum" hsizetype="Expanding" >
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <item>
                   <property name="text" >
                    <string>new</string>
                   </property>
                  </item>
                  <item>
                   <property name="text" >
                    <string>edit</string>
                   </property>
                  </item>
                  <item>
                   <property name="text" >
                    <string>view</string>
                   </property>
                  </item>
                  <item>
                   <property name="text" >
                    <string>execute</string>
                   </property>
                  </item>
                 </widget>
                </item>
                <item row="3" column="0" >
                 <widget class="QLabel" name="lParams" >
                  <property name="text" >
                   <string>Params</string>
                  </property>
                  <property name="wordWrap" >
                   <bool>false</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="WStackPage" >
           <property name="geometry" >
            <rect>
             <x>0</x>
             <y>0</y>
             <width>100</width>
             <height>30</height>
            </rect>
           </property>
           <layout class="QGridLayout" >
            <item row="0" column="0" >
             <widget class="QSEditor" native="1" name="eModule" >
              <property name="enabled" >
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="WStackPage" >
           <property name="geometry" >
            <rect>
             <x>0</x>
             <y>0</y>
             <width>100</width>
             <height>30</height>
            </rect>
           </property>
           <layout class="QGridLayout" >
            <item row="0" column="0" >
             <layout class="QHBoxLayout" >
              <item>
               <widget class="QLabel" name="textLabel1_4" >
                <property name="text" >
                 <string>File name</string>
                </property>
                <property name="wordWrap" >
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="eFileName" />
              </item>
              <item>
               <widget class="QPushButton" name="bFileName" >
                <property name="maximumSize" >
                 <size>
                  <width>20</width>
                  <height>32767</height>
                 </size>
                </property>
                <property name="text" >
                 <string>...</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="1" column="0" >
             <spacer>
              <property name="orientation" >
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeType" >
               <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="sizeHint" >
               <size>
                <width>20</width>
                <height>170</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="TabPage" >
       <attribute name="title" >
        <string>Aliases</string>
       </attribute>
       <layout class="QGridLayout" >
        <item row="0" column="0" >
         <widget class="Q3Table" name="tAliases" >
          <property name="numRows" >
           <number>0</number>
          </property>
          <property name="numCols" >
           <number>2</number>
          </property>
          <column>
           <property name="text" >
            <string>Lang</string>
           </property>
          </column>
          <column>
           <property name="text" >
            <string>Name</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <action name="Action" >
   <property name="text" >
    <string>Action</string>
   </property>
   <property name="iconText" >
    <string>Action</string>
   </property>
   <property name="name" stdset="0" >
    <string>Action</string>
   </property>
  </action>
  <action name="Action_2" >
   <property name="text" >
    <string>Action_2</string>
   </property>
   <property name="iconText" >
    <string>Action_2</string>
   </property>
   <property name="name" stdset="0" >
    <string>Action_2</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="5" margin="5" />
 <pixmapfunction>rcIcon</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>Q3GroupBox</class>
   <extends>QGroupBox</extends>
   <header>Qt3Support/Q3GroupBox</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Q3MainWindow</class>
   <extends>QWidget</extends>
   <header>q3mainwindow.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Q3WidgetStack</class>
   <extends>Q3WidgetStack</extends>
   <header>q3widgetstack.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Q3Table</class>
   <extends>Q3Frame</extends>
   <header>q3table.h</header>
  </customwidget>
  <customwidget>
   <class>Q3TextEdit</class>
   <extends>Q3Frame</extends>
   <header>q3textedit.h</header>
  </customwidget>
  <customwidget>
   <class>QSEditor</class>
   <extends>QWidget</extends>
   <header>qseditor.h</header>
  </customwidget>
 </customwidgets>
 <includes>
  <include location="local" >actiontree.h</include>
  <include location="local" >aliaseditor.h</include>
  <include location="local" >pixmappreview.h</include>
  <include location="local" >mainform.h</include>
  <include location="local" >qseditor.h</include>
 </includes>
 <resources/>
 <connections>
  <connection>
   <sender>cbType</sender>
   <signal>activated(int)</signal>
   <receiver>dEditAction</receiver>
   <slot>cbTypeChange()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bSetActive</sender>
   <signal>clicked()</signal>
   <receiver>dEditAction</receiver>
   <slot>bSetActive_clicked()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bRemoveActive</sender>
   <signal>clicked()</signal>
   <receiver>dEditAction</receiver>
   <slot>bRemoveActive_clicked()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bSetPassive</sender>
   <signal>clicked()</signal>
   <receiver>dEditAction</receiver>
   <slot>bSetPassive_clicked()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bRemovePassive</sender>
   <signal>clicked()</signal>
   <receiver>dEditAction</receiver>
   <slot>bRemovePassive_clicked()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbObject</sender>
   <signal>activated(int)</signal>
   <receiver>dEditAction</receiver>
   <slot>cbObjectChange()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
